### PR TITLE
Updated Documentation

### PR DESCRIPTION
**Pull request by CarterMcClellan**
Update to Tomcat v 9 (screenshot)

Update to Minio version 3.0.5 (screenshot)

Indicate IDE is for web development (Eclipse)

In AlbumDao.Java declare url or pass it directly (album.setUrl(minioClient.presignedGetObject(minioBucket, item.objectName(), 60 * 60 * 24))

### CODE REVIEW

**Review by deekoder — APPROVED** [review]

**Review by nitisht — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ We will use Eclipse IDE to build this example and include Jersey, JSON and asm p
  
 Name your project PhotoAPIService
 
-![minio_JAVA2](https://github.com/minio/minio-java-rest-example/blob/master/docs/screenshots/minio-JAVA2.jpg?raw=true)
+![TomCatv9](docs/screenshots/minio-TomCatv9.png)
```
Comment (deekoder) [inline README.md]: Needs to be full path. Otherwise images won't display in play.

File: README.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ This pom.xml will have all the configuration details that Maven needs, to build
 
  Here's the full pom.xml generated after adding all the above dependencies successfully. 
 
-![minio_JAVA5](https://github.com/minio/minio-java-rest-example/blob/master/docs/screenshots/minio-JAVA5.jpg?raw=true)
+![Dependencies](docs/screenshots/Minio-Dependencies.png)
```
Comment (deekoder) [inline README.md]: Needs to be full github path

**Review by deekoder — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ This pom.xml will have all the configuration details that Maven needs, to build
 
  Here's the full pom.xml generated after adding all the above dependencies successfully. 
 
-![minio_JAVA5](https://github.com/minio/minio-java-rest-example/blob/master/docs/screenshots/minio-JAVA5.jpg?raw=true)
+![Dependencies](https://github.com/minio/minio-java-rest-example/blob/master/docs/screenshots/Minio-Dependencies.png)
```
Comment (deekoder) [inline README.md]: ?raw=true is it needed. Maybe /maybe not. Pls confirm.

**Review by deekoder — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -104,7 +104,7 @@ This pom.xml will have all the configuration details that Maven needs, to build
 		<dependency>
 			<groupId>io.minio</groupId>
 			<artifactId>minio</artifactId>
-			<version>1.0.1</version>
+			<version>3.0.5</version>
```
Comment (deekoder) [inline README.md]: change to 3.0.6

**Review by deekoder — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -44,7 +44,7 @@ This pom.xml will have all the configuration details that Maven needs, to build
 
  Here's the full pom.xml generated after adding all the above dependencies successfully. 
 
-![minio_JAVA5](https://github.com/minio/minio-java-rest-example/blob/master/docs/screenshots/minio-JAVA5.jpg?raw=true)
```
Comment (harshavardhana) [inline README.md:47]: You should also remove this file from the repo.

File: README.md [hunk 6]
```diff
@@ -26,7 +26,7 @@ We will use Eclipse IDE to build this example and include Jersey, JSON and asm p
  
 Name your project PhotoAPIService
 
-![minio_JAVA2](https://github.com/minio/minio-java-rest-example/blob/master/docs/screenshots/minio-JAVA2.jpg?raw=true)
+![TomCatv9](https://github.com/minio/minio-java-rest-example/blob/master/docs/screenshots/minio-TomCatv9.png)
```
Comment (harshavardhana) [inline README.md]: Lets use tomcat v8.5.16.

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Pull request closed without merging**